examples/pwa/pwa.html

```diff
@@ -11,6 +11,33 @@
     if (window.location.pathname.indexOf('/examples') == 0) {
       window.location.assign('/pwa');
     }
+
+    function loadScript(url, callback) {
+        // Adding the script tag to the head as suggested before
+        var head = document.getElementsByTagName('head')[0];
+        var script = document.createElement('script');
+        script.type = 'text/javascript';
+        script.src = url;
+
+        // Then bind the event to the callback function.
+        // There are several events for cross browser compatibility.
+        script.onreadystatechange = callback;
+        script.onload = callback;
+
+        // Fire the loading
+        head.appendChild(script);
+    }
+
+    // Using this explicitly, as firefox errors due to our polyfills
+    loadScript("https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.0.3/webcomponents-sd-ce.js", function() {
+      console.log('Started loading webcomponents polyfill!');
+    });
+
+    setTimeout(function() {
```

Comment:
Normally, we'd load all scripts ASAP, but will block certain operations until ready. I know it's an example, but it's an important part nonetheless to do right. We also need to follow up with a relatively prescriptive documentation. Overall, we already know everything we need to make this the most performant operation, so we just need to do that :)